src/app/components/Curation/CurationPromo/index.test.tsx

```diff
@@ -92,12 +93,13 @@ describe('Curation Promo', () => {
   });
 
   describe('Live Promo', () => {
-    it('should display LiveLabel on a Live Promo', () => {
+    it('should display LiveLabel on a Live Promo without a timestamp present', () => {
```

Comment:
Should we separate out the two tests that are happening here in one 'it'?
The benefit of doing it the way done here is that it is quicker and more efficient, and it isn't like the code is hard to understand when you look at it.
The downside of having this as one test is that if you see 'should display LiveLabel on a Live Promo without a timestamp present' has failed, you don't know whether the failure is because of the live text is absent, or because the timestamp is present (you can see this when it shows you the line of code failed on of course, but not by the name of the test). If you separate the concerns, there are more tests to run, but the name of the failing test then tells you which failed: the live label's presense or the timestamp's absence.

In the hierarchical promo you already have the timstamp test as its own test.

Maybe you already discussed it and decided to do it this way for efficiency.

Reply:
That's a fair comment, and you're right @LilyL0u, we probably should do this for separation of concerns. We did talk about it, but now that you mention the fact that it might be hard to figure out what went wrong, then it probably is best to have a separate test. 

I would also say that the DRY (Don't Repeat Yourself) principle does not always need to apply to test code - often duplication is better as it makes tests more readable and easier to understand.

Reply:
Thanks! We have split this test into two and it also keeps it consistent with how we tested in the other test file for Hierarchical Grid.